wrangler-service/src/main/java/io/cdap/wrangler/service/directive/RecipeHandler.java

```diff
@@ -56,6 +57,13 @@ public class RecipeHandler extends AbstractWranglerHandler {
 
   private RecipeStore recipeStore;
 
+  // Injected by CDAP
+  @SuppressWarnings("unused")
+  private Metrics metrics;
```

Comment:
FYI - this `Metrics` object is injected by CDAP with dimensions "namespace", "app", "service", "run", "instance"